bin/setup

```diff
@@ -10,13 +10,15 @@ chdir APP_ROOT do
   step 'Installing dependencies' do
     system! 'gem install bundler --conservative'
     system('bundle check') || system!('bundle install')
+    system('which direnv') || system!('brew install direnv')
   end
 
   step 'Copying sample files' do
     ['.envrc'].each do |file|
       unless File.exist?(file)
         cp "#{file}.sample", file
         puts "Copied #{file}.sample to #{file}"
+        system! "ln -s #{file} .env"
```

Comment:
ahh I see. `citizen doctor` told me that I needed `direnv`, so if we want to stick with that perhaps we stay with `.envrc.sample` but also change the README to tell people to run `rails s` instead of `heroku local`?